cookbooks/ambari/test/integration/helpers/serverspec/spec_helper.rb

```diff
@@ -0,0 +1,8 @@
+require 'serverspec'
+
+if (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM).nil?
```

Comment:
Do we expect this to be run on Windows?

Reply:
No, I'm not expecting this to run on a windows system. This must have been auto-created. I can remove it if it causes any confusion.